deploy/deployment.yaml

```diff
@@ -89,3 +89,43 @@ spec:
                 secretKeyRef:
                   name: allocation-api-secrets
                   key: nomis_oauth_pub_key
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Does the exporter need to run as a separate deployment? I'd expect it to be running as a sidecar container in the same pod as the application, so that it can communicate with the application process via a socket and so that we can monitor each application instance independently 🤔

Reply:
Yes - that is correct. The exporter should be an additional container in the same pod. Good catch. Will make the change.

Reply:
I'd expect there to be times when it's useful to have separate application metrics for each replica as well - let's have a chat.

If we do go with this approach, don't we need some config to enable the exporter and application to communicate?